Documentation/gettingstarted/elasticsearch.rst

```diff
@@ -29,153 +34,154 @@ The file ``es-sw-app.yaml`` will deploy the Elasticsearch service which stores S
+
 
-    $ kubectl exec vader -- python get_search.py
-    Searching for Books by Darth Sidious
-    Got 2 Hits:
-    {'_index': 'sidious', '_type': 'tome', '_id': '2', '_score': 1.0, '_source': {'author': 'sidious', 'title': 'Force is Same for Dark Side and Jedi'}}
-    {'_index': 'sidious', '_type': 'tome', '_id': '1', '_score': 1.0, '_source': {'author': 'sidious', 'title': 'Why Convert a Jedi!'}}
-    Get Book 1 by Darth Sidious
-    {'author': 'sidious', 'title': 'Why Convert a Jedi!'}
+:: 
+
+    $ kubectl exec outpost -- python update.py 
+    Uploading Spaceship Diagnostics
+    {'_index': 'spaceship_diagnostics', '_type': 'stats', '_id': '1', '_score': 1.0, \
+     '_source': {'spaceshipid': '3459B78XNZTF', 'type': 'tiefighter', 'title': 'Engine Diagnostics', \
+                 'stats': '[OK] [ENGINE OK @SPEED 5000 km/s]'}}
 
 
 Step 4: Securing Elasticsearch Using Cilium
 ===========================================
 
-Fortunately for Darth Sidious, the Empire DevOps team is using Cilium for their Kubernetes cluster. Cilium provides L7 visibility and security policies to control Elasticsearch API access. In this case, Sidious orders to get the following policy pushed which gives him both ``GET`` and ``PUT`` access to his pods but restricts Vader's pods to only ``GET`` access.
 
-.. literalinclude:: ../../examples/kubernetes-es/es-sw-policy.yaml
+.. image:: images/cilium_es_gsg_topology.png
+   :scale: 40 %
 
+Following the least privilege security principle, we want to the allow the following legitimate actions and nothing more:
 
-Cilium follows the white-list, least privilege model for security. A *CiliumNetworkPolicy* contains a list of rules that define allowed requests, meaning that requests that do not match any rules are denied.
+* ``outpost`` service only has upload access to ``index: troop_logs``
+* ``spaceship`` service only has upload access to ``index: spaceship_diagnostics``
+* ``empire-hq`` service only has search access for both the indices 
 
-In this example, the policy rules are defined for inbound traffic (i.e., "ingress") connections to the *elasticsearch* service. Note that endpoints selected as backend pods for the service are defined by the *selector* labels. *Selector* labels use the same concept as Kubernetes to define a service. In this example, label ``'k8s:component': elasticsearch`` defines the pods that are part of the *elasticsearch* service in Kubernetes.
+Fortunately, the Empire DevOps team is using Cilium for their Kubernetes cluster. Cilium provides L7 visibility and security policies to control Elasticsearch API access. Cilium follows the **white-list, least privilege model** for security. That is to say, a *CiliumNetworkPolicy* contains a list of rules that define **allowed requests** and any request that does math the rules is denied. 
```

Comment:
does not match the rules ?

Reply:
@ianvernon what is not matching the rules?

Reply:
nevermind got it